### PR TITLE
SBM add check_identity_subscribed function

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr @imsickofmaps please review

File: test/test_utils.js [hunk 1]
```diff
@@ -1267,6 +1267,49 @@ describe("Testing app- and service call functions", function() {
                     .run();
             });
         });
+        describe("Testing check_identity_subscribed function", function() {
+            it("expect 'true'", function() {
```
Comment (gsvr) [inline test/test_utils.js]: Please make this more descriptive.

File: test/test_utils.js [hunk 2]
```diff
@@ -1267,6 +1267,49 @@ describe("Testing app- and service call functions", function() {
...
+                        utils.check_fixtures_used(api, [10, 23]);
+                    })
+                    .run();
+            });
+            it("expect 'false' (no subscription to messageset)", function() {
+                return tester
+                    .check(function(api) {
+                        return sbm.check_identity_subscribed("cb245673-aa41-4302-ac47-00000000001", "nurseconnect")
+                            .then(function(is_subscribed) {
+                                assert.ifError(is_subscribed);
```
Comment (gsvr) [inline test/test_utils.js]: Rather assert `false`

**Comment by gsvr** [comment]
Just make two improvements to the tests, then :+1:

**Pull request merged**